tools/setup-windows/SetupWindows.cs

```diff
@@ -40,31 +40,35 @@ public static int Main (string [] args)
 			var newTargets      = Path.Combine (prefix, "lib", "xamarin.android", "xbuild", "Xamarin", "Android");
 			var refAssemblies   = Path.Combine (progFiles, "Reference Assemblies", "Microsoft", "Framework", "MonoAndroid");
 			var newAssemblies   = Path.Combine (prefix, "lib", "xamarin.android", "xbuild-frameworks", "MonoAndroid");
+			var refAssembliesForVS = Path.Combine (vsInstall, "Common7", "IDE", "ReferenceAssemblies", "Microsoft", "Framework", "MonoAndroid");
```

Comment:
`vsInstall` may be `progFiles`, as per above (when `%VSINSTALLDIR%` isn't set), in which case this path won't really make sense. Should we actually create this link when `%VSINSTALLDIR%` isn't set?